enhancements/ingress/request-timeout-configuration.md

```diff
@@ -0,0 +1,231 @@
+---
+title: request-timeout-configuration
+authors:
+  - "@rfredette"
+reviewers:
+  - "@miciah"
+  - "@danehans"
+  - "@frobware"
+  - "@sgreene570"
+  - "@knobunc"
+  - "@miheer"
+  - "@candita"
+  - "@alebedev87"
+approvers:
+  - "@miciah"
+  - "@frobware"
+  - "@knobunc"
+creation-date: 2021-06-17
+last-updated: 2021-06-28
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced|informational
+see-also:
+replaces:
+superseded-by:
+---
+
+# Request Timeout Configuration
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This document proposes the addition of several API fields into the ingress
+controller API, allowing the cluster admin to adjust how long routers will hold
+open connections that are awaiting a response.
```

Comment:
```suggestion
open connections under various circumstances.
```
These settings include request deadline and client and server inactivity timeouts, so it's a broader set of options than just response deadline.